medmodels/matching/metrics.py

```diff
@@ -1,9 +1,11 @@
 """Metrics for comparing vectors in the context of matching classes."""
 
 import math
+from typing import Literal
 
 import numpy as np
 from numpy.typing import NDArray
+from typing_extensions import TypeAlias
```

Comment:
See https://github.com/limebit/medmodels/blob/main/medmodels/medrecord/types.py#L8-L14 on how to import this correctly